5.0/en/0x26-V17-WebRTC.md

```diff
@@ -37,10 +37,12 @@ In particular, it will be necessary to implement protections against flood attac
 
 Systems that rely solely on peer-to-peer media communication between web browsers, without the involvement of intermediate media servers, are excluded from these specific media-related security requirements.
 
+This section refers to the use of Datagram Transport Layer Security (DTLS). A requirement related to having a documented policy for management of cryptographic keys can be found in the "Cryptography" chapter. Information on approved cryptographic methods can be found either in the Cryptography Appendix of the ASVS or in documents such as NIST SP 800‑52 Rev. 2 or BSI TR‑02102‑2 (Version 2025‑01).
```

Comment:
maybe "This section refers to the use of Datagram Transport Layer Security (DTLS) in the context of WebRTC."?

Reply:
No, LGTM!

Reply:
Opened #3112